recipes/hugin/meta.yaml

```diff
@@ -0,0 +1,131 @@
+{% set version = "2019.2.0" %}
+{% set version_major_minor = ".".join(version.split('.')[:2]) %}
+
+package:
+  name: hugin
+  version: {{ version }}
+
+source:
+  #url: https://pilotfiber.dl.sourceforge.net/project/hugin/hugin/hugin-{{ version }}/hugin-{{ version }}.0.tar.bz2
+  url: https://iweb.dl.sourceforge.net/project/hugin/hugin/hugin-{{ version_major_minor }}/hugin-{{ version }}.tar.bz2
+  sha256: 8ba6bdfea246313f142f17f42e066c6888f51b72e4f8814b5e1c84ff56a95a3e
+  patches:
+    - windows_unbundle.patch
+
+build:
+  skip: true  # [osx]
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - libgomp      # [linux]
+    - make   # [unix]
+    - ninja  # [win]
+    - cmake
+    - swig
+    - binutils  # [linux]
+    - pkg-config  # [unix]
+    - {{ cdt('mesa-libgl-devel') }}  # [linux]
+    - {{ cdt('mesa-dri-drivers') }}  # [linux]
+    - {{ cdt('libselinux') }}  # [linux]
+    - {{ cdt('libxdamage') }}  # [linux]
+    - {{ cdt('libxxf86vm') }}  # [linux]
+    - {{ cdt('libxext') }}     # [linux]
+    # Just taken from wxWidgets
+    - {{ cdt('libice-devel') }}  # [linux]
+    - {{ cdt('libsm-devel') }}  # [linux]
+    - {{ cdt('libx11-devel') }}  # [linux]
+    - {{ cdt('libxcomposite-devel') }}  # [linux]
+    - {{ cdt('libxcursor-devel') }}  # [linux]
+    - {{ cdt('libxdamage-devel') }}  # [linux]
+    - {{ cdt('libxext-devel') }}  # [linux]
+    - {{ cdt('libxi-devel') }}  # [linux]
+    - {{ cdt('libxinerama-devel') }}  # [linux]
+    - {{ cdt('libxfixes-devel') }}  # [linux]
+    - {{ cdt('libxrandr-devel') }}  # [linux]
+    - {{ cdt('libxrender-devel') }}  # [linux]
+    - {{ cdt('libxtst-devel') }}  # [linux]
+    - {{ cdt('mesa-libEGL-devel') }}  # [linux]
+    - {{ cdt('mesa-libGL-devel') }}  # [linux]
+    - {{ cdt('xorg-x11-proto-devel') }}  # [linux]
+  host:
+    - xorg-libxfixes           # [linux]
+    - xorg-libsm               # [linux]
+    - python
+    - pip
+    - libtiff
+    - libpano13
+    - jpeg
+    - libpng
+    - openexr
+    # build 1017 has openexr support
+    # why is conda not picking this one up?
+    - vigra  1.11.1=*_1017
+    - exiv2
+    - glew
+    - sqlite
+    - lcms2
+    # Warning: With vanially conda, wxwidgets will not work on linux since they
+    # internally code strings as UTF-16 strings on windows
+    # conda-forge has patched conda: https://github.com/conda-forge/conda-feedstock/pull/117
+    # But upstream has rejected a similar patch:
+    # https://github.com/conda/conda/pull/9946
+    # In favor of more time to work on a longer term fix
+    # https://github.com/conda/conda-build/blob/master/tests/test_api_build.py#L240
+    - wxwidgets 3.1
+    - perl
+    - libglu  # [linux]
+    # boost was already migrated for vigra, and the build we
+    # need uses boost 1.72
+    - boost 1.72
+    - boost-cpp 1.72
```

Comment:
Can these be `>=1.72`?

Reply:
i was hoping to remove these eventually.